README.md

```diff
@@ -42,7 +46,13 @@ Options:
 
 
 
-[later we will include more details here]
+### Downstream Evaluation
+
+```
+python3 heareval/task_embeddings.py
```

Comment:
this won't work on a pip install. we want `python3 -m heareval.task_embeddings`

Should we create a separate 'runner' script to invoke generation of all the embeddings? That can accept the module and model weights from command line and run both task_embeddings and task_embeddings_with_labels